### ISSUE TITLE
UI on the pokemon statistic

### ISSUE THREAD

**Issue by TimRamandt** [TimRamandt/PoGoPvPData#15]
depends on #5

**Comment by TimRamandt** [comment]
Since it's also available in the daily view, it should be modulair

**Comment by TimRamandt** [comment]
Asked for feedback, and it is too confusing having both total encounters and lead in the same column, so I'll split it up which means filtering would be needed anymore

**Comment by TimRamandt** [comment]
removing the segmentation will be a separate issue for the next version
